roles/kerberos5/tasks/client.yml

```diff
@@ -0,0 +1,81 @@
+---
+
+- name: check required variables
+  fail: msg="{{ item }} is not defined"
+  when: item not in vars
+  with_items: "{{ krb5_cl_required_vars }}"
+
+- name: check if NetworkManager is present (ALTLinux)
+  command: >
+    rpm -q NetworkManager-daemon
+  register: networkmanager_present
+  failed_when: false
+
+- name: disable resolv.conf management by NM
```

Comment:
Btw, why do we do anything with  `resolv.conf`? We are not, like, configuring a local DNS or anything...

Reply:
I dunno. >_<

Reply:
Colleagues told me this is related to `systemd-networkd` distro configurations.